modules/COLL-01_Exploring-a-repository.yml

```diff
@@ -85,9 +89,13 @@ screens:
               id: COLL-01-qq4-o1
               response: "Good job! @crichID was the first person to comment on issue #2."
               correct: true
-            - value: # Null means it matches anything else - please make this the last option
+            - value: /hubyeti/i
```

Comment:
@kkemple same as above - are we leaving the option for either crichID or hubyeti to comment first?